src/cache-redis.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The Redis cache backend. See cache.js
+ *
+ * @prettier
+ */
+const Redis = require('redis');
+const config = require('./config.js');
+const client = Redis.createClient(config.redisURL);
+
+module.exports = {
+    async get(key) {
+        return new Promise(function(resolve, reject) {
+            client.get(key, (err, response) => {
+                if (err) {
+                    /* eslint-disable no-console */
+                    console.error('Error from Redis server:', err);
+                    /* eslint-enable no-console */
+                    resolve(null);
+                } else {
+                    resolve(response);
+                }
+            });
+        });
+    },
+
+    set(key, value) {
```

Comment:
I would vote for making this `async` as well, just like you did for the `get`. I'd prefer to have all I/O operations done asynchronously.